Makefile

```diff
@@ -120,6 +120,7 @@ uninstall: manifests kustomize
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy: manifests kustomize
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMAGE_SHA}
+	# $(KUSTOMIZE) build config/default > out.yaml
```

Comment:
Do we need to keep this line?

Reply:
No, but I've been using it during dev. But I'll remove the Makefile from this PR.